static/js/theme-map/VerticalFullPageMapOrchestrator.js

```diff
@@ -24,6 +25,11 @@ class VerticalFullPageMapOrchestrator extends ANSWERS.Component {
      */
     this._mapContainerSelector = '#js-answersMap';
 
+    /**
+     * The container in the DOM for the no results block.
+     */
+    this._noResultsContainer = document.querySelector('.js-answersNoResults');
```

Comment:
Can we query `this._container` instead of `document` to keep it scoped to this component?

Reply:
yes love that!